Drop-In/src/test/java/com/braintreepayments/api/dropin/DropInActivityUnitTest.java

```diff
@@ -527,6 +536,27 @@ public void anyExceptionExitsActivityWithError() {
         assertExceptionIsReturned("sdk-error", new Exception("Error!"));
     }
 
+    @Test
+    public void testAndroidPayVisible() {
```

Comment:
Thanks for the feedback. I ma not able to run the tests tho. AS complains  that AndroidManifest.xml is missing. I read in your wiki that you are still working to allow people to run the tests. Is that still the case? Please advise